app/uk/gov/hmrc/exports/controllers/SubmissionController.scala

```diff
@@ -38,20 +33,32 @@ import scala.concurrent.{ExecutionContext, Future}
 class SubmissionController @Inject()(
   authenticator: Authenticator,
   submissionService: SubmissionService,
+  declarationService: DeclarationService,
   headerValidator: HeaderValidator,
-  cc: ControllerComponents,
-  bodyParsers: PlayBodyParsers
+  cc: ControllerComponents
 )(implicit executionContext: ExecutionContext)
     extends RESTController(cc) with JSONResponses {
 
+  def create(id: String): Action[Unit] = authenticator.authorisedAction(parse.empty){ implicit request =>
```

Comment:
👍

Reply:
Not sure I'd do `Action[Unit]` usually a POST  endpoint would return the thing it created e.g. the `Submission`

Reply:
Type parameter in action is what action take and does not do anything with Result.

Good point it is nice to get what we created.

Reply:
Ok - this would result with returned submission - `submit` return `Future[Submission]` and then is lifted used `Created` status and using implicit from `RESTController` is automatically marshall it correct representation with `Created` status